scripts/linum_compute_attenuation.py

```diff
@@ -0,0 +1,70 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+""" Computes the tissue apparent attenuation coefficient map and then use the average
+attenuation to compensate its effect in the OCT reflectivity data.
+"""
+# TODO: Keep the OCT pixel format (which is float32 ?)
+import argparse
+
+import numpy as np
+from scipy.ndimage import gaussian_filter
+
+from linumpy.preproc.icorr import get_extendedAttenuation_Vermeer2013
+from linumpy.io.zarr import read_omezarr, save_zarr
+
+def _build_arg_parser():
+    p = argparse.ArgumentParser(
+        description=__doc__, formatter_class=argparse.RawTextHelpFormatter)
+
+    # Mandatory parameters
+    p.add_argument("input",
+                   help="A single slice to process (ome-zarr).")
+    p.add_argument("output",
+                   help="Output attenuation map (ome-zarr).")
+
+    # Optional argument
+    p.add_argument("-m", "--mask", default=None,
+                   help="Optional tissue mask (.ome.zarr)")
+    p.add_argument("--s_xy", default=0.0, type=float,
+                   help="Lateral smoothing sigma (default=%(default)s)")
+    p.add_argument("--s_z", default=5.0, type=float,
+                   help="Axial smoothing sigma (default=%(default)s)")
+
+    return p
+
+
+def main():
+    # Parse arguments
+    p = _build_arg_parser()
+    args = p.parse_args()
+
+    # Loading the data
+    zarr_vol, res = read_omezarr(args.input, level=0)
+    vol = np.moveaxis(zarr_vol, (0, 1, 2), (2, 1, 0))
```

Comment:
We could change the behavior of the attenuation estimation method to avoid having to swap the axes.

Reply:
Added todo in 912bbb4